tools/bin/ecflow/processes/main/models/create_global_file.py

```diff
@@ -22,16 +23,33 @@
 # read in the desired variables from the config file
 global_template = config_dict['VIC']['GlobalFileTemplate']
 global_file_path = config_dict['VIC']['GlobalFilePath']
-model_date = config_dict['VIC']['ModelDate']
+state_path = config_dict['VIC']['StatePath']
 
-# parse out year, month, and day from the model date, which has
+vic_start_date = config_dict['VIC']['vic_start_date']
+vic_end_date = config_dict['VIC']['vic_end_date']
+vic_save_state = config_dict['VIC']['vic_save_state']
+
+# parse out year, month, and day from the model dates, which have
 # the form YYYY-MM-DD
-model_year = model_date[:4]
-model_month = model_date[5:7]
-model_day = model_date[8:10]
+start_year = vic_start_date[:4]
```

Comment:
If it is a standard `YYYY-MM-DD` then you can simply use an existing date parser. For example:

``` python

from dateutil.parser import parse
datestr = '2000-03-04"
start = parse(datestr)
```
Then `start.year`, `start.month` and `start.day` have each of the date elements